locale/uk_UA/manager.po

```diff
@@ -710,6 +710,33 @@ msgstr "Введіть контактні дані, як правило, це г
 msgid "manager.setup.privacyStatement"
 msgstr "Заява про конфіденційність"
 
+msgid "manager.setup.dateTime"
+msgstr "Дата та час"
+
```

Comment:
@asmecher can we accept translations for non-English locales in PRs like this or should Vitalii go through Weblate?